src/almanack/metrics/metrics.yml

```
@@ -127,6 +137,38 @@ metrics:
       Count of the number of tags within the repository
       within the last 182 days from now (where now is a
       reference to table value of almanack-table-datetime).
+  - name: "repo-stargazers-count"
+    id: "SGA-GL-0014"
+    result-type: "int"
+    description: >-
+      Count of the number of stargazers on repository
+      remote hosting platform.
+  - name: "repo-uses-issues"
+    id: "SGA-GL-0015"
+    result-type: "bool"
+    description: >-
+      Whether the repository uses issues as a bug or
```

```suggestion
      Whether the repository uses issues (for example, as a bug and/or
```

This won't actually tell you the answer to "or" (it's just a boolean, right)?

---

Thanks, yes, better definition, I'll add this in!